### PR TITLE
Fix Python path finding for benchmark tests

### PR DESCRIPTION
**Pull request by pjh5**

### CODE REVIEW

File: .jenkins/caffe2/bench.sh [hunk 1]
```diff
@@ -13,7 +13,8 @@ else
     num_gpus=0
 fi
 
-cmd="$PYTHON $CAFFE2_PYPATH/python/examples/resnet50_trainer.py --train_data null --batch_size 64 --epoch_size 6400 --num_epochs 2"
+caffe2_pypath="$(dirname $(python -c 'import caffe2; print(caffe2.__file__)'))"
```
Comment (bddppq) [inline .jenkins/caffe2/bench.sh]: `__file__` is a list, do `__file__[0]`

**Review by bddppq — APPROVED** [review]
pre-approval

File: .jenkins/caffe2/bench.sh (hunk 1 above)
Comment (pjh5) [reply .jenkins/caffe2/bench.sh]: On the Linux docker I tested this on it is a string. I will investigate more

Comment (pjh5) [reply .jenkins/caffe2/bench.sh]: It looks like the behavior of __file__ was changed in 3.4 https://bugs.python.org/issue18416 . Maybe this is not a very safe method

Comment (bddppq) [reply .jenkins/caffe2/bench.sh]: Sorry I thought you were using `__path__`. `__file__` indeed is a string.

**Comment by bddppq** [comment]
bench run https://ci.pytorch.org/jenkins/job/caffe2-builds/job/py2-devtoolset7-rocmrpm-centos7.5-trigger-test/6434/

**Pull request closed without merging**